### DISCUSSION TITLE
conjugate heat transfer interface

### DISCUSSION THREAD

**Discussion by Krystalbbling** [idaholab/moose#26159] [Q&A Modules: Navier-Stokes]
Dear Expert，
Please what operations are required for the conjugate heat transfer interface using the e-file generated by the cubit software so that the grids of the fluid side and the solid side interface can be combined into one, because I have now encountered a problem, that is, the solid domain and the fluid domain run separately. It went smoothly, but it seems from the results that the temperature of the solid domain is not effectively transferred to the fluid domain. The fluid domain is constant temperature. I suspect that the interface on both sides is independent (but it seems that the nodes on both sides are corresponding) , which bothers me very much. I would like to ask you how to deal with the fluid-structure interaction interface when dealing with complex models (which cannot be automatically generated with Moose) to obtain effective solutions. 
Looking forward to your reply.
KRYSTAL

**Comment by GiudGiud** [comment]
Hello

Did you merge the two surfaces with the 'aligned nodes'? This is a common miss when using externally generated mesh file.
Can you please run the mesh diagnostics generator on your mesh?
https://mooseframework.inl.gov/moose/source/meshgenerators/MeshDiagnosticsGenerator.html
Guillaume
